rllib/policy/policy.py

```diff
@@ -45,7 +34,11 @@ class Policy(metaclass=ABCMeta):
     """
 
     @DeveloperAPI
-    def __init__(self, observation_space, action_space, config):
+    def __init__(self,
+                 observation_space,
+                 action_space,
+                 config,
+                 exploration=None):
```

Comment:
Why allow this to be specified in both config and argument? It would be cleaner to only instantiate it via config.

Reply:
Update here?